src/formElementPlugins/checkbox/src/ApplicationView.tsx

```diff
@@ -58,8 +58,8 @@ const ApplicationView: React.FC<ApplicationViewProps> = ({
   currentResponse,
   validationState,
 }) => {
-  const { getPluginStrings } = useLanguageProvider()
-  const strings = getPluginStrings('checkbox')
+  const { getPluginTranslator } = useLanguageProvider()
+  const t = getPluginTranslator('checkbox')
```

Comment:
This is the usage for plugins. It gets a custom version of `t` specific to the plugin, but subsequent usage is exactly the same, i.e. `t('KEY')`